electron-app/src/server/submission/parser/section-parsers/description.parser.ts

```diff
@@ -68,6 +68,10 @@ export class DescriptionParser {
           name: 'tags',
           content: website.generateTagsString(tags, description, websitePart),
         },
+        {
+          name: 'cw',
```

Comment:
q: So if no {cw} tag is put into the description, there will never be a content warning in the text body?

Reply:
Correct. Putting content warnings in the text body is explicitly done by the user if they want it there. If used, the user probably wants to prefix it with something anyway, such as `Content: {cw}` or  `(CW {cw})` or whatever.